src/components/Note/Note.jsx

```diff
@@ -1,44 +1,80 @@
 import React from 'react'
 
-function Note({ note }) {
-  return (
-    <form onSubmit="">
-      <label>
-        Name:
-        <input type="text" value={note.name} />
-      </label>
-      <label>
-        Distiller:
-        <input type="text" value={note.distiller} />
-      </label>
-      <label>
-        Age:
-        <input type="text" value={note.age} />
-      </label>
-      <label>
-        Category:
-        <select>
-          <option value={note.category}>{note.category}</option>
-        </select>
-      </label>
-      <label>
-        Color:
-        <input type="text" value={note.color} />
-      </label>
-      <label>
-        Country:
-        <input type="text" value={note.country} />
-      </label>
-      <label>
-        Rating:
-        <input type="number" value={note.rating} />
-      </label>
-      <label>
-        Tasting Notes:
-        <textarea cols="30" rows="10" value={note.tastingNotes} />
-      </label>
-    </form>
-  )
+class Note extends React.Component {
+  constructor(props) {
+    super(props)
+    const { note } = this.props
+    this.state = {
+      id: note.id,
+      name: note.name,
+      distiller: note.distiller,
+      age: note.age,
+      category: note.category,
+      color: note.color,
+      rating: note.rating,
+      country: note.country,
+      tastingNotes: note.tastingNotes
+    }
+
+    this.updateNote = this.updateNote.bind(this)
+  }
+
+  updateNote(event) {
+    const name = event.target.name
+    this.setState({ [name]: event.target.value })
+  }
+
+  render() {
+    return (
+      <form>
+        <label>
+          Name:
+          <input type="text" name="name" value={this.state.name} onChange={this.updateNote} />
+        </label>
+        <label>
+          Distiller:
+          <input
+            type="text"
+            name="distiller"
+            value={this.state.distiller}
+            onChange={this.updateNote}
+          />
+        </label>
+        <label>
+          Age:
+          <input type="text" name="age" value={this.state.age} onChange={this.updateNote} />
+        </label>
+        <label>
+          Category:
+          <select name="category" onChange={this.updateNote}>
+            <option value={this.state.category}>{this.state.category}</option>
```

Comment:
I think this is the one form control we need to fix so they are all working properly.
Right now there's only ever one option in the select box, and it changes depending on the value of the Note data given to it. 

We need an array of pre-defined categories, and the user should be able to select one from all those options (and we should keep track of that state).

Can you just define a hard-coded list and `map()` over the options? The one passed in on props should initially be `selected`, and then use the `value` attribute on the root `<select>` tag to keep track of changes to the selected one: https://reactjs.org/docs/forms.html#the-select-tag